doc/examples/gui/wxpython/frame.py

```diff
@@ -41,8 +41,8 @@
 from example.dialogs import ExampleMapDialog
 
 # It is possible to call grass library functions (in C) directly via ctypes
-# however this is less stable. Example is available in trunk/doc/python/, ctypes
-# are used in nviz, vdigit, iclass gui modules.
+# however this is less stable. Example is available in trunk/doc/examples/python/,
+# ctypes are used in nviz, vdigit, iclass gui modules.
```

Comment:
It's not from here, but I just noticed here: trunk refers to SVN right?

Reply:
Yes. I guess it's outdated and we should replace it. Out of this PR, though.